scripts/scene/BuildingManager.gd

```diff
@@ -102,6 +102,11 @@ func _on_building_state_input(event: InputEvent) -> void:
 func _on_selecting_roomtype_state_entered() -> void:
 	build_menu.show_room_panel(room_types)
 
+func _on_selecting_roomtype_state_input(event: InputEvent):
```

Comment:
apologies for adding unrelated code to the pr but while building i felt we needed to allow right click on selecting_roomtype to allow stop building as it works for all other steps